builder/init.go

```diff
@@ -36,17 +36,13 @@ func (b *Builder) getLatestUpstreamVersion() (string, error) {
 // initDirs creates the directories mixer uses
 func (b *Builder) initDirs() error {
 	// Create folder to store local rpms if defined but doesn't already exist
-	if b.Config.Mixer.LocalRPMDir != "" {
-		if err := os.MkdirAll(b.Config.Mixer.LocalRPMDir, 0777); err != nil {
-			return errors.Wrap(err, "Failed to create local rpms directory")
-		}
+	if err := os.MkdirAll(b.Config.Mixer.LocalRPMDir, 0777); err != nil {
```

Comment:
I'm not sure I like failing the entire init process if this fails, since before it could be circumvented (or never even seen) if a user did not set the local-rpm conf or pass the flag. Now it is a requirement that it gets made. Not sure how I would resolve that, maybe print a warning and keep going?

Reply:
Since the path is fixed on init and points to $pwd, this would also means that the creation of `local-bundles` and `upstream-bundles` folders would also fail (not to mention all the other artifacts created on $pwd during init).

Reply:
so hard-fail on `local-bundles` since that is the only thing that is technically necessary for a mix right?

Reply:
not really, you can have a mix with an empty `local-bundles`, so I assume you don't even need that folder. Still, you do need the other artifacts that are created on pwd (upstreamurl, upstreamversion, mixversion and mixer.state) so if if fails to create a folder it will also fail to create those files.

Reply:
We need to stop creating those files based on pwd and put them under the `VersionPath`

Reply:
Anyways, I don't see the hard failure as a big deal, I just wanted to bring it up.